extensions/wikia/AdEngine/js/template/bfaaMobile.js

```diff
@@ -54,10 +55,39 @@ define('ext.wikia.adEngine.template.bfaaMobile', [
 		adsModule.setSiteHeadOffset(height);
 	}
 
+	function animate(startAspectRatio, finalAspectRatio) {
+		var windowWidth = document.body.clientWidth;
+
+		adSlot.style.height = windowWidth / startAspectRatio + 'px';
+
+		setTimeout(function () {
```

Comment:
I wonder if we can get any cleaner solutions than `setTimeout(() => {}, 0)`. What other options have you considered?